server/api/src/test/java/org/hawkular/apm/server/api/model/zipkin/SpanTest.java

```diff
@@ -55,4 +65,62 @@ public void testPartialUrl() throws MalformedURLException {
         assertEquals(url.getPath(), result.getPath());
     }
 
+    @Test
+    public void testSerialization() throws IOException, ClassNotFoundException {
+        BinaryAnnotation binaryAnnotation = new BinaryAnnotation();
+        binaryAnnotation.setKey("http.method");
+        binaryAnnotation.setValue("GET");
+        binaryAnnotation.setEndpoint(createEndpoint("bonjour", "127.0.0.2", 8090));
+        binaryAnnotation.setType(AnnotationType.STRING);
+
+        Span span = new Span(Arrays.asList(binaryAnnotation));
```

Comment:
Would it be possible to have a test mapping config, with some more binary annotations, to ensure the endpoint or component type are filled in, as well as details and properties?